org.scala-ide.sdt.core.tests/test-workspace/scalajavadepjavaiswrong/src/test/J.java

```diff
@@ -0,0 +1,10 @@
+package test;
+
+public class J {
+	public static void main(String[] args) {
```

Comment:
Only problem is in IDE tabs in `test-workspace` are ok, but in console build they are not.